dotgov-domains/current-federal.csv

```diff
@@ -1,4 +1,4 @@
-Domain Name,Domain Type,Agency,Organization,City,State
+﻿Domain Name,Domain Type,Agency,Organization,City,State

```

Comment:
I think there was a change to this line that added an unexpected Unicode character at the start of the line. This can be seen when running `less` on the file in a Unix environment:

<img width="684" alt="screen shot 2018-05-30 at 12 02 31 pm" src="https://user-images.githubusercontent.com/4592/40732482-6879f5e2-6401-11e8-9577-0ca2951e9f62.png">

And the Pulse data loading process is crashing when loading domain data:

```
$ python -m data.processing
Traceback (most recent call last):
  File "/home/ubuntu/.pyenv/versions/3.6.5/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/ubuntu/.pyenv/versions/3.6.5/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/scan/pulse/data/processing.py", line 1180, in <module>
    run(None, options())
  File "/opt/scan/pulse/data/processing.py", line 110, in run
    domains, agencies, gathered_subdomains = load_domain_data()
  File "/opt/scan/pulse/data/processing.py", line 220, in load_domain_data
    dict_row[headers[i]] = cell
IndexError: list index out of range
```

I'll try to submit a PR to fix this issue.